app/src/main/java/org/liberty/android/fantastischmemo/ui/MiscTabFragment.java

```diff
@@ -86,6 +86,7 @@
     private View donateButton;
     private View helpButton;
     private View aboutButton;
+    private View favoriteButton;
```

Comment:
In the other favourite branch we write favorite with the following spelling: "favourite". Please change your spelling to that.

Reply:
Okay ill take this into account and change the spelling so it matches with the rest of the code when I work on this new merged branch